src/envoy/auth/jwt.cc

```diff
@@ -31,6 +31,7 @@
 namespace Envoy {
 namespace Http {
 namespace Auth {
+namespace {
 
 namespace Base64url {
```

Comment:
neither, since it is in `.cc` file, you don't need to namespace it, just remove the Base64url and Util namespace is find. have all of them in a unnamed namespace